src/azure/azure-gradle-step.yml

```diff
@@ -0,0 +1,21 @@
+parameters:
+  continueOnError: false
+  publishTestResults: false
+
+steps:
+  - bash: |
+      set -e
+      # when executing on an Azure Pipelines build agent (not in a container)
+      # default to java 11
+      if [ -n "${JAVA_HOME_11_X64}" ]; then
+        export JAVA_HOME="${JAVA_HOME_11_X64}"
+      fi
+      ./gradlew --version
+      ./gradlew --scan --warning-mode=all -Dplatform.tooling.support.tests.enabled=true build
+    displayName: Test
+    continueOnError: ${{ parameters.continueOnError }}
+  - task: PublishTestResults@2
+    displayName: Publish Test Results
+    inputs:
+      testResultsFiles: '**/build/test-results/test/TEST-*.xml'
```

Comment:
Wondering why this also catches the "expected to fail" reports from the "documentation" module...

...either we need to exclude those, or delete the reports before collecting test results.

Reply:
Actually `com.example.vintage.VintageTest` is from the platform-tooling-support-tests project.

@ethomson Can we exclude '**/build/test-workspace/**'?

Reply:
Indeed. If excluding doesn't work out, I'll delete those reports after `platform-tooling-support-tests` are performed.

/fup2 #1810

Reply:
@marcphilipp @sormuras Unfortunately `testResultsFiles` only takes wildcards and there's no exclusion pattern.  If `rm` is okay then that's the easiest path forward, tbqh.

Reply:
Already fixed via https://github.com/junit-team/junit5/commit/7f535408b182cb2ad2d5befe1a62c928439b987a